src/WebServer_HardwarePage.ino

```diff
@@ -75,6 +83,23 @@ void handle_hardware() {
 #ifdef FEATURE_SD
   addFormPinSelect(formatGpioName_output("SD Card CS"), "sd", Settings.Pin_sd_cs);
 #endif // ifdef FEATURE_SD
+#ifdef HAS_ETHERNET
+  addFormSubHeader(F("Ethernet"));
+  addRowLabel_tr_id(F("Ethernet PHY type"), "ethtype");
+  String ethPhyTypes[2] = { F("LAN8710"), F("TLK110") };
+  addSelector("ethtype", 2, ethPhyTypes, NULL, NULL, Settings.ETH_Phy_Type, false, true);
+  addFormNumericBox(F("Ethernet PHY Address"), "ethphy", Settings.ETH_Phy_Addr, 0, 255);
+  addFormNote(F("I²C-address of Ethernet PHY (0 or 1 for LAN8720, 31 for TLK110)"));
```

Comment:
This looks like non-ASCII, which will give issues on compiling.

Reply:
I don't expect any issues in neither displaying I2C nor compiling by PlatformIO :)

Reply:
> I don't expect any issues in neither displaying I2C nor compiling by PlatformIO :)

Well the thing is I have seen issues with non-standard ASCII characters in our code (even in comments)
Not sure whether different Git clients or their settings may cause issues, but the way how Arduino code is parsed is not always handling these correct.
PlatformIO should now show warnings when parsing code with non-ASCII characters, which was added when I had those issues last time.
Displaying is another thing. 
When you use characters from the extended ASCII table and not Unicode encoding, it depends on the used code table whether it will be displayed like you intended.
For example in countries with Cyrillic characters I can imagine they have something other than the 'superscript 2' for that character.

If you browse through the code, you can see all special characters are handled via either HTML-encoded Unicode, or HTML-escaped characters.

Reply:
Ahh, now i understood your intention. Ok, i've changed it to HTML entity &sup2; ;)
Should be even more readable than HTML coded character ;)